CHANGELOG.md

```diff
@@ -13,6 +13,7 @@
 - Use OpenTelemetry span name as fallback for transaction name ([#3557](https://github.com/getsentry/sentry-java/pull/3557))
   - In certain cases we were sending transactions as "<unlabeled transaction>" when using OpenTelemetry 
 - Add OpenTelemetry span data to Sentry span ([#3593](https://github.com/getsentry/sentry-java/pull/3593))
+- No longer selectively copy OpenTelemetry attributes to Sentry spans `data` ([#3663](https://github.com/getsentry/sentry-java/pull/3663))
```

Comment:
Should we also include the fact, that we no longer set the span data on the transaction data?